Makefile

```diff
@@ -1,18 +1,41 @@
-# One of:
-# libgultra_rom, libgultra_d, libgultra
-TARGET ?= libgultra_rom
-VERSION ?= L
-VERBOSE ?= 0
-
-# Use handwritten ASM implementations of select `gu` functions
-MGU_ASM ?= 1
+# Makefile to build libultra
 
 include util.mk
 
-ifeq ($(VERBOSE), 0)
-V=@
-else
-V=
+# Preprocessor definitions
+
+WORKING_DIR := $(shell pwd)
+
+DEFINES = MODERN_CC=1
+
+SRC_DIRS :=
```

Comment:
Is there a reason to declare this here or can it be moved down to where it's first assigned?

Reply:
Fixed by making MODERN_CC redundant